server/src/test/java/io/mantisrx/master/jobcluster/JobClusterTest.java

```diff
@@ -1611,9 +1611,12 @@ public void testJobSubmitWithNoSchedInfoUsesJobClusterValues() {
 
             assertEquals(SUCCESS, detailsResp.responseCode);
             assertEquals(JobState.Accepted, detailsResp.getJobMetadata().get().getState());
-            assertEquals(clusterLabels.size(),detailsResp.getJobMetadata().get().getLabels().size());
+            // The + 2 is for the two system labels that are always added
+            assertEquals(clusterLabels.size() + 2,detailsResp.getJobMetadata().get().getLabels().size());
```

Comment:
the `+2` could be a constant somewhere so it doesn't get out of sync as more system labels are added.

Reply:
Yep makes sense